### PR TITLE
Use `fields @message` for --return

### LINKED ISSUE #1: Improve `--return` behavior
Currently we use `stats earliest(@message)` to return log message.
https://github.com/mackerelio-labs/check-aws-cloudwatch-logs-insights/blob/1f3c76124192ff1327989ff04b939b7bf7a17775/lib/app.go#L174

But we don't need such `stats` and can simple use `@message` so that we can more messages matched by the filter.

### PR DESCRIPTION
**Pull request by astj**
Closes #1

now we can simplly use returned `@message` as source of `--return`.

### CODE REVIEW

File: lib/app.go [hunk 1]
```diff
@@ -184,7 +185,7 @@ func (p *awsCWLogsInsightsPlugin) startQuery(startTime, endTime time.Time) (*str
 		StartTime:     aws.Int64(startTime.Unix()),
 		LogGroupNames: aws.StringSlice(p.LogGroupNames),
 		QueryString:   aws.String(p.fullQuery()),
-		Limit:         aws.Int64(1),
+		Limit:         aws.Int64(10),
```
Comment (astj) [inline lib/app.go:188]: With this change, now `--return` will return up to 10 log messages matched by the query.

**Comment by lufia** [comment]
question: What is it meaning that there is `Closes #4` in the description?

#4 is the merged pull request

**Comment by astj** [comment]
Sorry, it's wrong. this aims to close #1.

**Review by lufia — APPROVED** [review]
LGTM!

**Pull request merged**